ci/infra/jobs/build_amis/dw-al2-concourse-ami.yml

```diff
@@ -0,0 +1,92 @@
+resources:
+  - name: dw-al2-concourse-ami
+    .: (( inject meta.resources.ami-builder-configs ))
+    source:
+      paths:
+        - dw-al2-concourse-ami/*
+        - generic_packer_template.json.j2
+
+  - name: dw-al2-concourse-ami-pr
+    .: (( inject meta.resources.ami-builder-configs-pr ))
+    source:
+      paths:
+        - dw-al2-concourse-ami/*
+        - generic_packer_template.json.j2
+
+jobs:
+  - name: dw-al2-concourse-ami
+    plan:
+      - get: dw-al2-concourse-ami
+        trigger: true
+      - .: (( inject meta.plan.get-al2-hardened-ami ))
+      - .: (( inject meta.plan.generate-manifest ))
+        input_mapping:
+          source_ami: dw-al2-hardened-ami
+        config:
+          inputs:
+            - name: source_ami
```

Comment:
because you are using input_mapping, this should move to meta (around line 65) and be removed from each of the jobs

Reply:
I didn't create that part,  general-ami does not use input mapping, so this may break it

trying the suggestion creates:
```
error: error unmarshaling JSON: while decoding JSON: malformed task step: json: cannot unmarshal object into Go struct field TaskStep.input_mapping of type string
```

Reply:
sounds like you missed a hyphen

Reply:
This is done.